pyproject.toml

```diff
@@ -23,10 +23,12 @@ pytest = "^7.3.2"
 Sphinx = "^6.1.3"
 taskipy = "^1.11.0"
 pyproject-flake8 = "^6.0.0.post1"
-install = "^1.3.5"
```

Comment:
I found that `poetry install` failed since `install` package does not exist as of September 2024.
Please refer https://github.com/eugenekolo/pip-install/issues/4 .